Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN apk update && \
     -s /bin/bash \
     -u 1000 \
     -G git \
-    git
+    git && \
+  echo "git:$(date +%s | sha256sum | base64 | head -c 32)" | chpasswd
```

Comment:
No, we need to set a password otherwise OpenSSH complains that the user is locked, e.g.

```
User git not allowed because account is locked
input_userauth_request: invalid user git
```

And since we are using Alpine we only got the pretty limited `adduser` from Busybox.